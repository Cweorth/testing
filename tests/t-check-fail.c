#include <cut.h>

TEST(checkFail) {
    CHECK(0);
    CHECK(0);
}
