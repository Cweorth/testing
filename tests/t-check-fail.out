[  1] checkFail............................................................[1;31mFAIL[0m
    check '0' (check-fail.c:4)
    check '0' (check-fail.c:5)


Summary:
  tests:       1
  succeeded:   0
  skipped:     0
  failed:      1
